docs/index.md

```diff
@@ -1598,7 +1598,7 @@ tests as shown below:
   <head>
     <meta charset="utf-8" />
     <title>Mocha Tests</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
`initial-scale` defines a ratio -- hence, the number written in "float/double" format.

Reply:
...which is fine an integer in this case.